pvacseq/lib/call_iedb.py

```diff
@@ -46,12 +52,12 @@ def main(args_input = sys.argv[1:]):
     url = prediction_class_object.url
 
     response = requests.post(url, data=data)
-    if response.status_code == 500:
-        #Retry once
+    retries = 0
+    while response.status_code == 500 and retries < args.iedb_retries:
         response = requests.post(url, data=data)
-        if response.status_code == 500:
-            #Retry a second time
-            response = requests.post(url, data=data)
+        print("Retries %s" % retries)
```

Comment:
It might be nice if this were a little more descriptive, e.g.:
`"IEDB: Retry %s of %s" % (retries, args.iedb_retries)`

Now that it might happen 100 times in a row, should this introduce a short delay before attempting again?